spring-cloud-gcp-pubsub/src/test/java/org/springframework/cloud/gcp/pubsub/reactive/PubSubReactiveFactoryTests.java

```diff
@@ -163,11 +195,13 @@ private void setUpMessages(String... messages) {
 				switch (nextPayload) {
 					case "stop":
 						return result;
-					case "throw":
+					case "timeout":
```

Comment:
Could you update this method's javadoc to separate "throw" from "timeout"? This was a fairly convoluted setup to begin with; if someone has to troubleshoot it, I'd rather they have all the detail for what this method is supposed to do.

Reply:
Of course! some how I totally didn't see the javadoc...

Reply:
I updated the javadoc. If it is not clear enough, then please let me know.